backend/src/index.ts

```diff
@@ -0,0 +1,27 @@
+import fastify from 'fastify'
+import * as db from "./dbutil";
+
+const server = fastify()
+
+server.get('/waterheater', function (request, reply) {
```

Comment:
Let’s just have a single “/appliances” endpoint instead of segmenting it by type. We can add in request params later but for now Ricky can do any necessary filtering on the front end.